lib/tasks/republish.rake

```diff
@@ -4,9 +4,11 @@ require "csv"
 require "services"
 
 namespace :republish do
-  # Caution: These tasks should be avoided in production environnments  as
-  # they change the date information of draft documents and extra superfluous
-  # history to published documents.
+  # Do not run these tasks in production, unless absolutely necessary,
```

Comment:
Perhaps keeping "Caution:" or something like at the beginning would catch the eye :) Or perhaps a env flag one must set e.g. `DO_SOMETHING_DANGEROUS=true`

Reply:
I thought about a flag, but decided that we'd probably only want the flag in production.

It's fine to do in integration and staging, and if the flag is required *everywhere* there's the temptation to just copy and paste the command from integration to production without really thinking about it.